MakeHist/HTLT.C

```diff
@@ -98,4 +128,100 @@ void HTLT(){
 
     leg_HTLT->Draw();
 
+    c1->SaveAs("./hists/"+prefix+"_HTLT.png");
+
+
+    //make bkg for siginificance plot
+    HTLT_DB->Add(HTLT_ST);
+    HTLT_DB->Add(HTLT_TT);
+    HTLT_DB->Add(HTLT_DY);
+
+    TH1F * HTLT_DB_copy = (TH1F*) HTLT_DB->Clone("DB_copy");
+
+    TH1F * HTLT_200_copy = (TH1F*) HTLT_200->Clone("200_copy");
+
+    //std::cout << "ComputeIntegral " << HTLT_DB->ComputeIntegral() << "\n";
+    float previousBin;
+    float currentBin;
+    int nBins = HTLT_DB->GetNbinsX();
+
+    //std::cout << "right cut \n";
+    for(int i=-2;i<nBins+3;i++){
+
+        previousBin = HTLT_DB->GetBinContent(i-1);
+        currentBin = HTLT_DB->GetBinContent(i);
+
+        HTLT_DB->SetBinContent(i, currentBin+ previousBin);
+        //std::cout << HTLT_DB->GetBinContent(i) << "\n";
+
+        previousBin = HTLT_200->GetBinContent(i-1);
+        currentBin = HTLT_200->GetBinContent(i);
+
+        HTLT_200->SetBinContent(i, currentBin+ previousBin);
+        //std::cout << HTLT_200->GetBinContent(i) << "\n";
+
+
+
+    }
+
+    std::cout << "left cut \n";
+    for(int i=nBins+2;i>-2;i--){
```

Comment:
Again, explain the range of the loop and behavior of the Get/Set bin content when going out of range.